docs-new/docs/contribute/docs/spell-check.md

```diff
@@ -0,0 +1,76 @@
+# Spell checker
+
+All PRs that are pushed to a Keptn repository
+are run through a spell checker that is based on the
+[check-spelling](https://github.com/check-spelling/check-spelling)
+GitHub action.
+
+Note, however, that you are still responsible for reading your text carefully.
+The tool does not flag if you used a real word that is not the right word,
+if you misuse "setup" versus "set up", and so forth.
+
+## Handling Spell Checker errors
+
+If you get a Spell Checker error:
+
+1. Right-click the `details` link for Spell Checker.
```

Comment:
right click..?

Reply:
Google style guide uses right-click.  But we can deviate, of course.

Reply:
That's fine, it's just missing some context. Where do I right click?
In any case I don't think we need to be this specific. I think it's enough to tell people to check the spell checker github actions job summary.

Reply:
How does one access the spell checker github actions job summary?

Reply:
See if what I have is accurate